sycl/test/fpga_tests/compute_units.cpp

```diff
@@ -0,0 +1,78 @@
+//==------------ compute_units.cpp - SYCL FPGA compute units test ----------==//
```

Comment:
You need to add compile and run string for the test. You can take one from pipes.cpp

Reply:
Thanks!